abdiff/config.py

```diff
@@ -31,6 +33,18 @@ def webapp_port(self) -> int:
         port = self.WEBAPP_PORT or "5000"
         return int(port)
 
+    @property
+    def transmogrifier_max_workers(self) -> int:
+        """Maximum number of Transmogrifier containers to run in parallel."""
+        max_workers = self.TRANSMOGRIFIER_MAX_WORKERS or 6
+        return int(max_workers)
+
+    @property
+    def transmogrifier_timeout(self) -> int:
+        """Timeout for a single docker container."""
+        timeout = self.TRANSMOGRIFIER_TIMEOUT or 60 * 60 * 5  # 5 hours default
```

Comment:
Established docker container timeout via env vars, and defaults to **5 hours**.  Until we speed up Transmogrifier, in a whole different body of work, a large `alma` or `dspace` file can take 2+ hours.